build/optaplanner-build-parent/pom.xml

```diff
@@ -32,7 +32,7 @@
     <version.com.fasterxml.jackson.core>2.12.1</version.com.fasterxml.jackson.core>
     <version.com.h2database>1.3.173</version.com.h2database>
     <version.com.thoughtworks.xstream>1.4.16</version.com.thoughtworks.xstream>
-    <version.io.quarkus>1.11.0.Final</version.io.quarkus>
+    <version.io.quarkus>1.13.0.Final</version.io.quarkus>
```

Comment:
There is a whole bunch of other dependencies in this POM, which may be overriding their versions coming from master.
Did you also check if those need to be bumped too?

Reply:
In the long run, we're thinking of starting by importing a quarkus bom (question is which one, not the platform bom of course as that contains optaplanner), so we don't get to decide our hibernate version any more, nor have to keep it in sync.